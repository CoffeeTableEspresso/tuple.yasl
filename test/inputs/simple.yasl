const tuple = require("tuple")

echo tuple(undef, 1, 2.3, .str, true)

