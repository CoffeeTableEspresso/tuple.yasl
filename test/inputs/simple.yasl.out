tuple(undef, 1, 2.3, str, true)
